main/zoe/api/zoe-helpers.md

```diff
@@ -173,9 +173,44 @@ import { makeZoeHelpers } from '@agoric/zoe/src/contractSupport/zoeHelpers';
 
 const { swap } = makeZoeHelpers(zoe);
 
-  const seat = harden({
-    matchOffer: () => swap(firstInviteHandle, inviteHandle),
-  });
+  // `firstOfferHandle` is from a prior offer to the contract
+  const hook = newHandle => swap(firstInviteHandle, newHandle);
+  return zcf.makeInvitation(hook);
+```
+
+## zoeHelper.inviteAnOffer(options)
+- `options` `{offerHook, customProperties, expected}` Optional name 
+  properties of the 
+- Returns: a promise for the new inviteHandle
+
+Make an invitation to submit an Offer to this contract. This
+invitation can be given to a client, granting them the ability to
+participate in the contract.
+
+If "offerHook" is provided, it will be called when the invitation is exercised
```

Comment:
There's two new terms here - "invitation" and "exercised". I think readers will be confused. We've talked about invites being redeemed (not necessarily better language) so I think we should stick with invite and redeemed for consistency.

Reply:
Done